lib/go-tc/servers.go

```diff
@@ -700,6 +700,44 @@ type CommonServerProperties struct {
 	XMPPPasswd       *string              `json:"xmppPasswd" db:"xmpp_passwd"`
 }
 
+type CommonServerPropertiesV40 struct {
```

Comment:
I like option `a` since it is simple and easy to understand. Unless @rawlinp is really hung on option `b`, I will not be changing the logic.

Reply:
I leave the decision up to you, Rima -- I generally prefer option B, but I'm not hung up on it. Since the plan is to eventually cleanly remove API v3, the duplication is temporary (or, _should_ be temporary at least).

Reply:
Ok, Thanks @rawlinp. I will leave it as is. And I updated my earlier comment. So, unless it is the right thing to do, I will not be changing the logic.